services/web/server/src/simcore_service_webserver/storage_api.py

```diff
@@ -85,3 +87,27 @@ async def delete_data_folders_of_project_node(
     )
 
     await _delete(session, url)
+
+
+async def is_healthy(app: web.Application) -> bool:
+    client, api_endpoint = _get_storage_client(app)
```

Comment:
make sure this also has a very low timeout

Reply:
same as above